src/app/rundown-view/components/rundown-header/rundown-header.component.ts

```diff
@@ -22,9 +22,9 @@ export class RundownHeaderComponent implements OnInit, OnDestroy, OnChanges {
   public rundown: Rundown
 
   public showStyleVariant?: ShowStyleVariant
-  public currentLocalDate: Date = new Date()
-  private timeResolutionIntervalId: ReturnType<typeof setInterval>
-  private showStyleVariantSubscription?: SubscriptionLike
+  public currentLocalDate: number
+  private showStyleVariantSubscription?: Subscription
+  private rundownTimingContextSubscription?: Subscription
   public readonly IconButton = IconButton
   public readonly IconButtonSize = IconButtonSize
```

Comment:
Would be nice if we didn't mix public and private variables but instead kept the public variables together and the private variables together.

Reply:
They are now ordered.